luckeysherpa07/koraCounter
# Added Algorithm For with Demo Co-ordinate

## Issue to close
- close #16 

## Branch dependency
- No dependency

## Changes
- Added algorithm with Demo Co-ordinate

## Checklist
- [ ] Assigned yourself?
- [ ] Added reviewer(s)?
- [ ] Spec added for new feature?
- [ ] Spec passed?
- [ ] Tested in a shared environment?
- [ ] Postman request added if new endpoint's implemented?
- [ ] Conflict resolved ?

## Steps to test
1. Build and Run
2. Check on console

## Points to reviews
- things reviewer need to look into

## Evidence of fix if necessary

### Before
- screenshot, log or something

### After
- screenshot, log or something
